drivers/interrupt_controller/Kconfig.plic

```diff
@@ -10,3 +10,14 @@ config PLIC
 	help
 	  Platform Level Interrupt Controller provides support
 	  for external interrupt lines defined by the RISC-V SoC.
+
+if PLIC
```

Comment:
Hi! Thanks for this PR.

I'm wondering - could we generalize the PLIC shell to an Interrupt Controller shell, similar to what other drivers do? This would essentially mean that two new symbols would have to be implemented for each arch:
* `arch_irq_stat_get`
* `arch_irq_stat_reset`

And the generic driver in `drivers/interrupt_controller/intc_shell.c` would use them. Since this would be an opt-in feature, we wouldn't have to implement the support for it in every interrupt controller that is present in Zephyr right away.

Reply:
Each of the interrupt controllers is quite unique and doesn't have a unified interface like `i2c`, `flash`... I'd rather keep this within PLIC for now

Reply:
Fair enough :)